src/course_bot/localization.clj

```diff
@@ -0,0 +1,19 @@
+(ns course-bot.localization
```

Comment:
What is the idea behind this refactoring?

Just for me, i18n naming is better.

Reply:
This refactor was needed to avoid cyclic dependencies. For example, if you want to use tr method in talk and talk's methods in general class.